CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## master
 
 * Add your own contributions to the next release on the line below this with your name.
+- [fix] Fix PINMemoryCache to check if TTL before updating last fetched date: [#222](https://github.com/pinterest/PINCache/pull/222) (jerelevine)
```

Comment:
Should this be just "check TTL"? The "if" seems out of place.

Reply:
Ok

Reply:
Removed the unnecessary "if"